src/framework/net/httplogin.cpp

```
@@ -44,27 +45,27 @@ void LoginHttp::cancel() {
 void LoginHttp::Logger(const auto& req, const auto& res) {
-    std::cout << "======= LOG ======= " << std::endl;
-    std::cout << "-- REQUEST --" << std::endl;
-    std::cout << req.method << std::endl;
-    std::cout << req.path << std::endl;
-    std::cout << req.body << std::endl;
+    g_logger.debug("======= HTTP LOG =======");
+    g_logger.debug("-- REQUEST --");
+    g_logger.debug("{}", req.method);
+    g_logger.debug("{}", req.path);
+    g_logger.debug("{}", req.body);
 
```

Cppcheck (2.19.0)

[error] `#error` "Compiler not supported."

(preprocessorErrorDirective)